aten/src/THC/THCNumerics.cuh

```diff
@@ -205,7 +205,7 @@ struct THCNumerics<at::Half> {
   static inline __host__ __device__ at::Half erf(at::Half a) { return ::erf(a); }
   static inline __host__ __device__ at::Half erfc(at::Half a) { return ::erfc(a); }
   static inline __host__ __device__ at::Half erfinv(at::Half a) { return ::erfinv(a); }
-  static inline __host__ __device__ at::Half abs(at::Half a) { return ::abs(a); }
+  static inline __host__ __device__ at::Half abs(at::Half a) { return std::abs(a); }
```

Comment:
Is this a recent change in the compiler? I remember `std::` was not supported before.

Reply:
It's still in general the case that `std::` math functions are not supported and I do not see much indication of that changing any time soon. However, in this particular instance hcc prefers us to use `std::abs()` for the call, i.e., it warns about the `::abs()` usage.